package.json

```diff
@@ -7,11 +7,11 @@
     "postinstall": "lerna bootstrap",
     "build": "./Build/createVersionFile.sh && cross-env NODE_ENV=production yarn build:ui",
     "build:ui": "NEOS_BUILD_ROOT=$(pwd) webpack --progress --colors",
-    "build:ui:watch": "yarn build:ui -- --watch",
-    "build:ui:watch-poll": "yarn run build:ui -- --watch-poll --watch",
+    "build:ui:watch": "yarn build:ui watch",
```

Comment:
`--watch` is needed, otherwise there is an error when running `yarn build:ui:watch`